shared/video-sdk/authentication-workflow/project-implementation/cpp.mdx

```diff
@@ -100,33 +119,27 @@
     with:
 
     ```cpp
-    if (token == "")
-    {
-        // Fetch new token 
-        token = fetchToken(serverUrl, channelName, tokenRole, uid, expireTime);
-        if (token == "")
-        {
-            AfxMessageBox(L"Invalid Token.");
-            return;
-        }
-    }
-    if (0 == agoraEngine->joinChannel(token.c_str(), channelName.c_str(), NULL, uid))
-    {
-        isJoin = true;
-    }
+    CString channel;
+	GetDlgItemText(IDC_EDIT1, channel);
```

Comment:
No need , As I am already setting Channel name from Agora console on `CAgoraImplementationDlg` class.. I have not made any change from GetStarted doc for Channel Name.